cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/ReadSegmentRangeCommand.java

```diff
@@ -42,29 +45,43 @@ public ReadSegmentRangeCommand(CommandArgs args) {
     }
 
     @Override
-    public void execute() throws ExecutionException, InterruptedException, TimeoutException {
-        ensureArgCount(4);
+    public void execute() throws ExecutionException, InterruptedException, TimeoutException, IOException {
+        ensureArgCount(5);
```

Comment:
Please, add some precondition checks on the offset/length arguments (e.g., offset >= 0, length >=0)

Reply:
Added the precondition checks.